tensorflow_datasets/scripts/documentation/templates/dataset_doc_template.py

```diff
@@ -0,0 +1,382 @@
+"""Dataset catalog documentation template.
+
+Displayed in https://www.tensorflow.org/datasets/catalog/.
+
+"""
+import textwrap
+import collections
+import tensorflow_datasets as tfds
+
+_NIGHTLY_DOC_UTIL = None
+_VISU_DOC_UTIL = None
+
+
+# --------------------------- Builder sections ---------------------------
+
+
+def display_description(builder):
+  return f"""\
+*   **Description**:
+
+{builder.info.description}
+"""
+
+
+def display_config_description(builder):
+  if builder.builder_config:
+    return f"""\
+*   **Config description**: {builder.builder_config.description}
+"""
+  return ""
+
+
+def display_homepage(builder):
+  return f"""\
```

Comment:
Formatting of `"""` looks bad. Would it be possible to wrap all of those in `tfds.core.utils.dedent` which apply `textwrap.dedent` + `strip` to normalize formatting.